pkg/bundle/setup/clustersetup.sh

```
@@ -228,11 +228,11 @@ patch_default_route() {
 
 set_credentials() {
     pr_info  "setting cluster credentials"
-    podman run --rm -ti ghcr.io/crc-org/crc-cloud/httpd-tools:latest developer $PASS_DEVELOPER > htpasswd.developer
+    podman run --rm ghcr.io/crc-org/crc-cloud/httpd-tools:latest developer $PASS_DEVELOPER > htpasswd.developer
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)